openid-4-verifiable-presentations-1_0.md

```diff
@@ -136,6 +136,11 @@ Implementations can also be built on top of OpenID Connect Core, which is also b
 
 Any of the OAuth 2.0 related specifications, such as [@RFC9126] and [@RFC9101], and Best Current Practice (BCP) documents, such as [@RFC8252] and [@I-D.ietf-oauth-security-topics], can be implemented on top of this specification.
 
+In summary, OpenID for Verifiable Presentations is a framework that requires profiling
```

Comment:
Shouldn't this text also mention that a profile needs to select a certain credential format?

Reply:
yes, addressed below.